src/lib/Components/Browse.php

```diff
@@ -32,26 +32,13 @@ protected function getLocationId()
         return "false";
     }
 
-    protected function getLocationRestId()
-    {
-        if ($this->request->attributes->get('location')) {
-            return $this->urlGenerator->generate(
```

Comment:
removing that makes the `urlGenerator` useless, so it should not be injected any more.